lib/ably/version.rb

```diff
@@ -1,6 +1,6 @@
 module Ably
   VERSION = '1.1.7'
-  PROTOCOL_VERSION = '1.1'
+  PROTOCOL_VERSION = '1.2'
```

Comment:
As discussed at the previous standup we're not at API version 1.2 yet so I don't agree with this change.